test/walkman-test.el

```diff
@@ -148,11 +150,15 @@
 
 (ert-deftest walkman--test-at-point ()
   (cl-letf (((symbol-function 'walkman--exec)
-             (lambda (args &optional keep-headers)
+             (lambda (args)
                (walkman-response--create :code 200 :status "OK" :headers '() :body ""))))
-    (with-temp-buffer
-      (insert-file-contents "test/sample-request")
-      (should (not (walkman-at-point))))))
+    (let ((response (with-temp-buffer
```

Comment:
Don't understand what this test is doing – what is the `(should (not` testing for? Can't we test that we get back the expected status code, headers and body?

Reply:
I'm not sure where you see a `(should (not`...

Reply:
here:

![image](https://user-images.githubusercontent.com/2683004/100553752-906d7b80-3287-11eb-971b-8173baef9086.png)

If the updated test looks OK then all good! :slightly_smiling_face: